assets/config.js

```diff
@@ -13,8 +13,28 @@ export default {
 	loadDbName: "starter_wp",
 	apiBase: "/wp-json/wp/v2/",
 	contactEmail: "wecount@inclusivedesign.ca",
-	socialMedia: [],
-	funders: [],
+	// Using raw SVG content is to work around the issue with dynamically loading and injecting inline SVGs into the template.
```

Comment:
Another approach we can look into for the future is to `<use>`. This will allow you to externally reference an SVG and still be able to style it. However, it does not work in IE 11 and would need a fall back for that. The benefit, other than cleaner looking HTML files, is that you benefit from browser caching.

This [article](https://tympanus.net/codrops/2015/07/16/styling-svg-use-content-css/) has a very detailed explanation of using `<use>` and styling SVGs. Perhaps we should file an issue for this.

Reply:
Umm... By reading the article, I don't think it will help with solving the issue with dynamically loading SVGs. The syntax to use `<use>` is:

```
<symbol id="my-icon" viewBox="0 0 30 30">
	<!-- icon content / shapes here -->
</symbol>

<use xlink:href="#my-icon" x="100" y="300" />
```
It requires the icon content to be preloaded into `<symbol>`. We'll hit the same loading issue here.

Please correct me if my understanding is wrong. Thanks.

Reply:
`<symbol>` is defined in the svg sprite. It is used to group the svg shapes so they can be referenced externally by `<use>`. The href in `<use>` would specify the svg sprit file and the # would reference the particular symbol. I've written this up on the [Research SVG Icons](https://wiki.fluidproject.org/display/fluid/Research+SVG+Icons) wiki page when I was looking into this stuff several years ago. It seems like we're getting to a point where support in most browsers could actually make this a viable solution. At the time I was researching this there were issues with styling elements.

Reply:
I will file an issue for this when this pull request is merged so the issue can use links to the code in the master branch.